app/dashboard/notifications.py

```diff
@@ -238,6 +238,43 @@ def maybe_market_tip_to_slack(tip, event_name):
     return True
 
 
+def get_status_header(bounty):
+    statuses = ['Open']
+    status = bounty.status
+    if status == 'unknown':
+        return ""
+    elif status == 'cancelled':
+        statuses.append('**Cancelled**')
+    elif status == 'expired':
+        statuses.append('**Expired**')
+    else:
+        if status == 'open':
+            statuses = ['**Open**', 'Started', 'Submitted', 'Done']
+        elif status == 'started':
+            statuses.append('**Started**')
```

Comment:
This could be: `statuses += ['**Started**', 'Submitted', 'Done']`

Reply:
updated